plugin/src/main/java/net/dzikoysk/funnyguilds/config/transformer/formatting/DoubleRangeFormattingTransformer.java

```diff
@@ -0,0 +1,60 @@
+package net.dzikoysk.funnyguilds.config.transformer.formatting;
+
+import eu.okaeri.configs.schema.GenericsPair;
+import eu.okaeri.configs.serdes.BidirectionalTransformer;
+import eu.okaeri.configs.serdes.SerdesContext;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.dzikoysk.funnyguilds.config.range.formatting.DoubleRangeFormatting;
+
+public class DoubleRangeFormattingTransformer extends BidirectionalTransformer<String, DoubleRangeFormatting> {
+
+    private static final Pattern RANGE_PATTERN = Pattern.compile("(-?[0-9.*]+)-(-?[0-9.*]+)");
+
+    @Override
+    public GenericsPair<String, DoubleRangeFormatting> getPair() {
+        return this.genericsPair(String.class, DoubleRangeFormatting.class);
+    }
+
+    @Override
+    public DoubleRangeFormatting leftToRight(String data, SerdesContext serdesContext) {
+        String[] split = data.split(" ");
+
+        Matcher matcher = RANGE_PATTERN.matcher(split[0]);
+
+        double min = Integer.MIN_VALUE;
+        double max = Integer.MAX_VALUE;
+        if (matcher.matches()) {
+            min = parseDouble(matcher.group(1), Integer.MIN_VALUE);
+            max = parseDouble(matcher.group(2), Integer.MAX_VALUE);
+        }
+
+        return new DoubleRangeFormatting(min, max, split[1]);
+    }
+
+    @Override
+    public String rightToLeft(DoubleRangeFormatting data, SerdesContext serdesContext) {
+        return data.toString();
+    }
+
+    private static double parseDouble(String numberString, double fallback) {
+        try {
+            if (numberString.contains("*")) {
+                return fallback;
+            }
+            else {
+                if (numberString.contains(".")) {
```

Comment:
Dlaczego nie wystarczy po prostu Double.parseDouble, po co ten fallback do parseInt

Reply:
Żeby potem w configu zamiast 4.0 było 4 ale gdy jest potrzebne po przecinku to 4.0 dalej działało...